backend/app/services/ai_agent_bridge.py

```
@@ -0,0 +1,431 @@
+        logger.info(
+            "ai_agent_bridge.started base_url=%s", self._settings.IOT_RELAY_BASE_URL
+        )
+
+    async def stop(self) -> None:
+        """앱 종료 시 호출. 현재 연결 끊고 task 취소."""
+        self._stop_event.set()
+        if self._client is not None:
+            try:
+                await self._client.aclose()
+            except Exception:  # noqa: BLE001
+                pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)